### PR TITLE
Added Verdant

### PR DESCRIPTION
**Pull request by ignoreintuition**
Submitting this PR means you have:

- [x] searched the [readme][readme-url] file and searched the [open pull requests][pr-url] (no duplicates please)
- [x] and will follow [the desired format][format-url]
- [x] used image links that resolve correctly, are appropriately sized, and are served over HTTPS

Once your PR is created, it will trigger a build a GitHub Action (Awesome-Lint). This includes a format linting process, so if there are any issues with the build, please address them.

Do you have further questions? Check the [contribution guidelines][contributing-url] or [open an issue][new-issue-url].

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

[pr-url]: https://github.com/edm00se/awesome-board-games/pulls
[readme-url]: https://github.com/edm00se/awesome-board-games/edit/main/readme.md
[format-url]: https://github.com/edm00se/awesome-board-games/blob/main/formatting.md
[contributing-url]: https://github.com/edm00se/awesome-board-games/blob/main/contributing.md
[new-issue-url]: https://github.com/edm00se/awesome-board-games/issues/new
#269

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -2222,6 +2222,20 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
...
 
+### [Verdant](https://www.flatout.games/verdant)
+
+> Verdant is a puzzly spatial card game for 1 to 5 players. You take on the role of a houseplant enthusiast trying to create the coziest interior space by collecting and arranging houseplants and other objects within your home. You must position your plants so that they are provided the most suitable light conditions and take care of them to create the most verdant collection.
+
+Each turn, you select an adjacent pair of a card and token, then use those items to build an ever-expanding tableau of cards that represents your home. You need to keep various objectives in mind as you attempt to increase plant verdancy by making spatial matches and using item tokens to take various nurture actions. You can also build your "green thumb" skills, which allows you to take additional actions to care for your plants and create the coziest space!
+
+Verdant has beautful artwork from Beth Sobel and was designed by Flatout games who also created Cascadia and Calico.
+
+![Verdant](https://cf.geekdo-images.com/c1XGn5ryzbqgMOHzObZpQw__original/img/RWppBKjLU5v6Pl-uW6IIxkCD4U4=/0x0/filters:format(png)/pic6094080.png)
```
Comment (edm00se) [inline readme.md:2233]: The image used is massive, please update it to one appropriately sized. We target ~250px in either height or width, whichever is the larger.

File: readme.md [hunk 2]
```diff
@@ -2222,6 +2222,20 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
 |-------: | -------: | ------:|
 |   2 - 4 |       5  |  20m   |
 
+### [Verdant](https://www.flatout.games/verdant)
+
+> Verdant is a puzzly spatial card game for 1 to 5 players. You take on the role of a houseplant enthusiast trying to create the coziest interior space by collecting and arranging houseplants and other objects within your home. You must position your plants so that they are provided the most suitable light conditions and take care of them to create the most verdant collection.
+
+Each turn, you select an adjacent pair of a card and token, then use those items to build an ever-expanding tableau of cards that represents your home. You need to keep various objectives in mind as you attempt to increase plant verdancy by making spatial matches and using item tokens to take various nurture actions. You can also build your "green thumb" skills, which allows you to take additional actions to care for your plants and create the coziest space!
+
+Verdant has beautful artwork from Beth Sobel and was designed by Flatout games who also created Cascadia and Calico.
```
Comment (edm00se) [inline readme.md:2231]: These should also be a part of the same markdown quote block.

```suggestion
> Each turn, you select an adjacent pair of a card and token, then use those items to build an ever-expanding tableau of cards that represents your home. You need to keep various objectives in mind as you attempt to increase plant verdancy by making spatial matches and using item tokens to take various nurture actions. You can also build your "green thumb" skills, which allows you to take additional actions to care for your plants and create the coziest space!
>
> Verdant has beautful artwork from Beth Sobel and was designed by Flatout games who also created Cascadia and Calico.
```

**Review by edm00se — CHANGES_REQUESTED** [review]
Please update the description block and the image.

File: readme.md (hunk 1 above)
Comment (ignoreintuition) [reply readme.md]: Replaced with one 250 x 250

File: readme.md (hunk 2 above)
Comment (ignoreintuition) [reply readme.md]: Updated

**Review by edm00se — APPROVED** [review]
refs #269

**Pull request merged**